packages/bundler/test/RIP7560Mode.test.ts

```diff
@@ -0,0 +1,96 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { spy } from 'sinon'
+import { assert } from 'chai'
+
+import { BundlerServer } from '../src/BundlerServer'
+import { MethodHandlerERC4337 } from '../src/MethodHandlerERC4337'
+import { MethodHandlerRIP7560 } from '../src/MethodHandlerRIP7560'
+import { initServer } from '../src/modules/initServer'
+import { connectContracts } from '../src/runBundler'
+import { resolveConfiguration } from '../src/Config'
+import { DebugMethodHandler } from '../src/DebugMethodHandler'
+import { parseEther } from 'ethers/lib/utils'
+import { OperationRIP7560, sleep } from '@account-abstraction/utils'
+
+describe.skip('RIP7560Mode', function () {
+  describe('running bundler in the RIP-7560 mode', function () {
+    let ethersProviderSpy: any
+    let bundlerServer: BundlerServer
+    let operationRIP7560: OperationRIP7560
+
+    // TODO: 'runBundler' is a spaghetti but this code is mostly copied from there - refactor to reuse
+    before(async function () {
+      const { config, provider, wallet } = await resolveConfiguration({
+        // todo: add default values for all that
+        beneficiary: '',
+        gasFactor: '1',
+        minBalance: '1',
+        mnemonic: './mnemonic.txt',
+        network: '',
+        maxBundleGas: 0,
+        autoBundleInterval: 0,
+        autoBundleMempoolSize: 0,
+        useRip7560Mode: true
+      })
+      const signer = provider.getSigner()
+
+      // we stub the ValidationManagerRIP7560 so the values of these fields do not matter
+      operationRIP7560 = {
+        accessList: undefined,
+        nonce: '0x0',
+        sender: '0x0000000000000000000000000000000000000000',
+        callGasLimit: '0x0',
+        maxFeePerGas: '0x0',
+        maxPriorityFeePerGas: '0x0',
+        factoryData: '0x',
+        chainId: '0x0',
+        value: '0x0',
+        signature: '0x',
+        verificationGasLimit: '0x0',
+        paymasterVerificationGasLimit: '0x0',
+        paymasterPostOpGasLimit: '0x0',
+        builderFee: '0x0',
+        callData: '0x',
+        paymasterData: '0x'
+      }
+
+      // fund deployment of the EntryPoint contract
+      await signer.sendTransaction({ to: await wallet.getAddress(), value: parseEther('1') })
+
+      const {
+        entryPoint
+      } = await connectContracts(wallet, config.entryPoint)
```

Comment:
7560 bundler doesn't have an entryPoint. does it?

Reply:
Fixed.